src/6_branch.js

```diff
@@ -67,16 +67,21 @@ var wrap = function(parameters, func, init) {
 			 * @type {function(?Error,?): undefined}
 			 */
 			var done = function(err, data) {
-				if (err && parameters === callback_params.NO_CALLBACK) {
-					throw err;
-				}
-				else if (parameters === callback_params.CALLBACK_ERR) {
-					callback(err);
+				try {
+					if (err && parameters === callback_params.NO_CALLBACK) {
+						throw err;
```

Comment:
The expectation for several calls is to throw an error if the Branch SDK has not been initialized. Further down from the `done` method, there are conditionals which check for various states: init pending, init failed, or not inited at all.  Each of those states passes the error to the `done` method, so that it can either be passed to a callback function if one exists, or thrown on line 72 if there is no callback.  The default callback for some methods defined in 6_branch.js also throws the error passed to it, so in lines 75 and 78, there is potential for an error being thrown.  But in all cases, the SDK User expects those errors in the specific cases where initialization fails.